src/iasm.c

```diff
@@ -413,6 +413,8 @@ struct OPND
     }
 };
 
+static OPND* const emptyOPND = (OPND*)(-1);
```

Comment:
Ah you're using null now for failed parses. Please switch the logic, errors are rare, missing operands are normal, but testing for null is slightly cheaper than comparing with -1.
So just introduce errorOPND = -1, which also fits how we do/did that in a couple of other places of the compiler. That also leaves room for more error codes.